juspay/hyperswitch
# fix(router): Remove requires_customer_action status to payment confirm

## Type of Change



## Description

This will allow the payment to be confirmed if the status is RequiresCustomerAction.

### Additional Changes






## How did you test it?


1.    Create a payment which will move to `RequiresCustomerAction`.
<img width="1041" alt="Screen Shot 2023-07-06 at 12 49 28 PM" src="https://github.com/juspay/hyperswitch/assets/131388445/fdc19689-45b8-4568-b835-a7f0f5a98e99">

2. Confirm the payment again
<img width="930" alt="Screen Shot 2023-07-06 at 12 49 39 PM" src="https://github.com/juspay/hyperswitch/assets/131388445/84392fa7-165a-4795-b8e4-f74f63729e9c">



## Checklist
